### PR TITLE
Wet, Wild, and Political: Shoredress & Stardress Technology For All!

### PR DESCRIPTION
**Pull request by Skies-Of-Blue**
## About The Pull Request

Removes the slick skin check on the wetsuit component, allowing Azulean technology from making Non-Akulans wet.

In Plain English: All "Shoredress" items (Akulan Jumpsuits + Hydro-Vaporizer) will now make you (yes, you!) wet when worn.

## Why it's Good for the Game

Players are somewhat limited in their ability to create aquatic characters who are _dependent_ on water, considering wet-stacks fade relatively quickly. Giving them more freedom of expression is a good thing! It was always a little weird that these items discriminated based on species, anyways.

## Proof of Testing

<details>
<summary>Screenshots</summary>
Derrik here was more than willing to take some photos for us. Just look at how he glistens and shines!

derrik
derrik 2

</details>

## Changelog

:cl:
balance: After a great deal of protest amongst the aquatically inclined species of the galaxy, the Azulean Monarchy has finally given in and removed the genetic lock on proprietary Shoredress and Stardress technology. Rejoice, folk of fin and gill, as hydro-vaporizers now provide succor for your parched skin.
/:cl:

### CODE REVIEW

**Comment by Skies-Of-Blue** [comment]
Hold on let me cook a minute.

**Comment by Skies-Of-Blue** [comment]
Okay I cooked.

**Comment by KnigTheThrasher** [comment]
i think you can modularize most of this stuff

**Comment by Skies-Of-Blue** [comment]
Should be about as modularized as it can be! Ready for review.
